packages/website/pages/docs/reference/deploy-contract.md

```diff
@@ -0,0 +1,87 @@
+# Deploy a Contract
```

Comment:
why do we need to ways to deploy with thirdweb? because it seems like a redundancy to have this in reference (also it is the wrong abstraction, I believe this is a "guide"). is it possible to consolidate this all under the thirdweb "tag" in the single deploy a contract guide?

Reply:
also I see there are many links and steps. i am curious: is this maintainable over time? is it possible it's better suited to reference official thirdweb documentation so we aren't maintaining it in two places (where on Taiko's site it could become deprecated?)

Reply:
yes basically what i would recommend is this. whatever is the quickest and shortest number of steps to get the user deploying a contract on thirdweb is recommended. and then i would recommend not adding anything to the reference section. so ideally, the only place where we are mentioning thirdweb is in the "deploy a contract" guide, under the thirdweb tab. 

for example, we have this line "For additional information on Deploy, please reference [thirdweb’s documentation(opens in a new tab)](https://portal.thirdweb.com/deploy)." which is added to the "Deploy a contract" guide. this could be a link to the documentation.

we could expand this by adding any additional useful links to thirdweb documentation. it's going to be important to give the minimum viable documentation for users to get to "hello world" with thirdweb, because then we can minimize the chance of us maintaining stale documentation, and always refer users to the latest thirdweb documentation. in this spirit, we also don't want to duplicate documentation, for the same risk of it going stale. instead, we would like to point to links to thirdwebs official documentation.

---

if this is confusing or blocking in any way, please feel free to add me as a friend on discord and reach out directly: `d1onys1us` is my username. would be happy to pair on this together over a call :)

Reply:
do you need any help here?

Reply:
Yeah- added you on Discord!

Reply:
hey @d1onys1us, I have made the changes!

Reply:
looks great, thanks so much @avneesh0612 !